src/ck-pics/picstreenode.C

```diff
@@ -214,6 +234,12 @@ bool Condition::test(double *input) {
   return ret;
 }
 
+/**
+ * PICS suggested solutions.
+ * Applied to code iff PICS is used for tuning otherwise just prints a suggestion
```

Comment:
This function never applies a solution to code, does it? I think this comment is a little misleading, it should clearly describe the behavior of just _this function_, along with some other context if necessary. Where do the solutions get applied?